src/mappings/multiTokens/events/approved.ts

```diff
@@ -60,10 +61,18 @@ export async function approved(
     const address = u8aToHex(data.owner)
 
     if (data.tokenId !== undefined) {
-        const tokenAccount = await ctx.store.findOneOrFail<TokenAccount>(TokenAccount, {
+        const tokenAccount = await ctx.store.findOne<TokenAccount>(TokenAccount, {
```

Comment:
I would rather keep `findOneOrFail` and let the indexer crash than report it, 
because even if it was a small mistake you will have to do a full re-sync as this will always result in corrupt data.

Reply:
I remember this being a @Bradez request at the end of the year. Though I partially agree with you we gotta consider that we are not always available to fix and having all systems to stop (wallet/transactions/nftio/notifications/etc) seems to be worse than having corrupted data. And putting all fatal alerts we will know and we will also know that it will require a resync after the fix.